tests/test_modeling_flax_common.py

```diff
@@ -167,7 +178,8 @@ def test_equivalence_pt_to_flax(self):
                 fx_outputs = fx_model(**prepared_inputs_dict).to_tuple()
                 self.assertEqual(len(fx_outputs), len(pt_outputs), "Output lengths differ between Flax and PyTorch")
                 for fx_output, pt_output in zip(fx_outputs, pt_outputs):
-                    self.assert_almost_equals(fx_output, pt_output.numpy(), 4e-2)
+                    if not isinstance(fx_output, tuple):  # TODO(Patrick, Daniel) - let's discard use_cache for now
```

Comment:
Think we don't really need this no? I would always disable `use_cache` in PyTorch here instead `pt_model.config.use_cache = False` and leave the test as it is. In Flax the cache should not be returned as a default (only if `init_cache` is called which it is not here I think)